databases/week2/lesson2_homework_part3.sql

```diff
@@ -0,0 +1,30 @@
+-- You should do these queries on the database hyf_lesson2, which we created last class.
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+USE hyf_lesson2;
+
+SELECT *
+FROM user
+WHERE email LIKE '%@spotify.com';
+
+-- Get all the tasks for 'Donald Duck' with status 'Not started'
+SELECT *
+FROM user
+INNER JOIN user_task ON user.id = user_task.user_id
+INNER JOIN task ON user_task.task_id = task.id
+INNER JOIN STATUS ON task.status_id = status.id
+WHERE user.name = 'Donald Duck'
+AND status.name = 'Not started';
```

Comment:
Please use * only when necessary. In this case as you are joining 4 tables, then the out put will be a table of 15+ columns. What is asked here that get tasks fro the specific user with a status of 'Not started'

Line 14, table name `STATUS` is not the same as `status` when it comes to table names.

Reply:
Yes, it makes sense. 👍🏻